onward/app/controllers/package.scala

```diff
@@ -0,0 +1,13 @@
+import slices.FixedContainers
+
+package object controllers {
```

Comment:
We got two different patterns for this, personally I prefer Mixins (e.g. https://github.com/guardian/frontend/blob/master/common/app/implicits/ItemResponses.scala) to package objects.

So this would be (for example) `implicits.ContainerSizes`